### PR TITLE
refactor: Use PNPM

### PR DESCRIPTION
**Pull request by avaly**
Replace `yarn` with [`pnpm`](https://pnpm.io/)

### CODE REVIEW

**Review by vanstinator — COMMENTED** [review]
Worth getting internal consensus one more time? It's been several months since we talked about converting the internal repos and that PR is still in draft.

File: .gitignore [hunk 1]
```diff
@@ -5,7 +5,6 @@
 node_modules
 tests/*/dist
 tests/*/package-lock.json
-tests/*/yarn.lock
 yarn-error.log
```
Comment (CarlosLozanoHealthCaters) [inline .gitignore:8]: @avaly We could remove all yarn assets from gitignore:
* yarn-error.log
* .pnp.*
* .yarn/*
* !.yarn/patches
* !.yarn/plugins
* !.yarn/releases
* !.yarn/sdks
* !.yarn/versions

**Review by CarlosLozanoHealthCaters — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ This section will walk you through how to get started working with the code.
 
 Verify that node.js is [installed](https://nodejs.org/en/download/).
 
-Verify that Yarn v1 is [installed](https://classic.yarnpkg.com/en/docs/install).
+Verify that pNPM is [installed](https://pnpm.io/installation).
```
Comment (CarlosLozanoHealthCaters) [inline CONTRIBUTING.md:17]: A typo here. **P**NPM

Comment (avaly) [reply CONTRIBUTING.md:17]: I swear I saw this somewhere on their website, but I can't find it anymore. I'll update.

**Review by vanstinator — APPROVED** [review]

**Pull request merged**